frontend/src/components/RecipeComments/RecipeComments.js

```diff
@@ -0,0 +1,67 @@
+import React, { useState } from 'react';
+import map from 'lodash/map';
+import { createComment } from '../../lib/apiClient';
+import './RecipeComments.css';
+
+const RecipeComments = ({ comments, recipeId, user }) => {
+  const [comment, setComment] = useState('');
+  const handleChange = (event) => {
+    event.preventDefault();
+    setComment(event.target.value);
+  };
+
+  const handleSubmit = () => {
+    createComment(comment, recipeId);
```

Comment:
You might need to pass `event` to the function and call `event.preventDefault();` above this to prevent some strange behavior.

Reply:
Agreed. If you want more context, the accepted answer to this stackoverflow should be helpful:
https://stackoverflow.com/questions/28479239/setting-onsubmit-in-react-js

(Same deal as with your `handleChange` function).